src/l-tooltip.js

```diff
@@ -4,11 +4,14 @@ import { tooltip } from "leaflet";
 import { tooltipConnected } from "./events";
 
 class LTooltip extends HTMLElement {
-  static observedAttributes = ["content"];
+  static observedAttributes = ["content", "permanent", "direction"];
 
   constructor() {
     super();
-    this.tooltip = tooltip();
+    this.tooltip = tooltip({
+      permanent: this.hasAttribute("permanent") ,
+      direction: this.getAttribute("direction")
```

Comment:
The default value for direction is `'auto'`, my usual approach is to omit the key from the options object if the attribute isn't on the element.

Reply:
yup, leaflet seems to handle nulls ok, but I'll implement the optional approach.